### DISCUSSION TITLE
MS Learn documentation is extremely outdated

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [CommunityToolkit/Windows#334] [Feedback]
The entire [Windows Community Toolkit Documentation](https://learn.microsoft.com/windows/communitytoolkit/) is outdated, with the vast majority of the latest changes in 2021 or 2022.

**Comment by michael-hawker** [comment]
@Jay-o-Way the new docs are here: https://learn.microsoft.com/en-us/dotnet/communitytoolkit/windows/

We haven't been able to redirect them yet, it's something I'm going to have @Arlodotexe dive into in March now that we've finally got past the RID .NET 8 updating... 🎉

I had also noticed the new docs have double stems for some things like `segmented/segmented`, so we need to look at that too and try and clean things up.

**Reply by michael-hawker** [reply]
I also need to check if the doc export script I wrote was checked in anywhere or if I have to grab it off my old machine... 🙈
